.kokoro/nightly/create-versions-csv.sh

```diff
@@ -57,7 +57,7 @@ echo "Inserting client_library_versions.cloud_java_client_library_release_dates.
 head  cloud_java_client_library_release_dates.csv
 echo "===================="
 
-bq load --autodetect --project_id=cloud-java-metrics --source_format=CSV \
+bq load --skip_leading_rows=1 --project_id=cloud-java-metrics --source_format=CSV --null_marker="-"\
```

Comment:
According to https://cloud.google.com/bigquery/docs/schema-detect#loading_data_using_schema_auto-detection, the `--skip_leading_rows` feature is only relevant when combined with `--autodetect`. 

Is it doing anything for us here?

Reply:
On my local tests, removing `--skip_leading_rows=1` gives throws me an error.
local log for comparison:
```
╭─~/repos/java-cloud-bom/libraries-release-data ‹fix-create-versions●› 
╰─$ bq load  --project_id=cloud-java-metrics --source_format=CSV --null_marker="-" \                       
cloud-java-metrics:client_library_versions.cloud_java_client_library_release_dates  \
cloud_java_client_library_release_dates.csv
Upload complete.
Waiting on bqjob_r79e0bb74027424df_0000018ca7eb35f3_1 ... (0s) Current status: DONE   
BigQuery error in load operation: Error processing job 'cloud-java-metrics:bqjob_r79e0bb74027424df_0000018ca7eb35f3_1': Error while reading data, error message: CSV processing encountered too many errors, giving up. Rows: 15211; errors: 1; max
bad: 0; error percent: 0
Failure details:
- Error while reading data, error message: CSV processing encountered
too many errors, giving up. Rows: 15211; errors: 1; max bad: 0;
error percent: 0
- Error while reading data, error message: Unable to parse;
line_number: 1 byte_offset_to_start_of_line: 0 column_index: 1
column_name: "release_date" column_type: DATE value: "release_date"
╭─~/repos/java-cloud-bom/libraries-release-data ‹fix-create-versions●› 
╰─$ bq load  --skip_leading_rows=1  --project_id=cloud-java-metrics --source_format=CSV --null_marker="-" \                                                                                                                                        1 ↵
cloud-java-metrics:client_library_versions.cloud_java_client_library_release_dates  \
cloud_java_client_library_release_dates.csv
Upload complete.
Waiting on bqjob_r6d7b9f40c4132137_0000018ca7eb7797_1 ... (1s) Current status: DONE   
```

The documentation does read confusing to me. But the line "Also consider providing an explicit schema in this case, so that you can assign column names." let me think the alternative to skip this row is to explicitly define schema in this load command with  `--schema` flag.